### PR TITLE
style(Sidebar) added ToolTip

### PR DESCRIPTION
**Pull request by berry-13**
## Summary

added tooltip sidebar support

### when sidebar is open

Screenshot 2023-10-09 231155

### when sidebar is close

Screenshot 2023-10-09 231201

## Change Type

- [x] Accessibility

## Testing

check if it properly works

## Checklist

- [x] My code adheres to this project's style guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented in any complex areas of my code
- [x] I have made pertinent documentation changes
- [x] My changes do not introduce new warnings
- [x] I have written tests demonstrating that my changes are effective or that my feature works
- [x] Local unit tests pass with my changes
- [x] Any changes dependent on mine have been merged and published in downstream modules.

### CODE REVIEW

File: client/src/components/Nav/Nav.tsx [hunk 1]
```diff
@@ -161,78 +163,95 @@ export default function Nav({ navVisible, setNavVisible }) {
...
-                      pageNumber={pageNumber}
-                      pages={pages}
-                      nextPage={nextPage}
-                      previousPage={previousPage}
-                      setPageNumber={setPageNumber}
-                    />
-                  </div>
+    <TooltipProvider delayDuration={300}>
+      <Tooltip>
+        <>
```
Comment (danny-avila) [inline client/src/components/Nav/Nav.tsx]: remove empty brackets (React Fragment), not needed when there is a parent (Tooltip)

File: client/src/components/Nav/Nav.tsx [hunk 2]
```diff
@@ -161,78 +163,95 @@ export default function Nav({ navVisible, setNavVisible }) {
...
-        </div>
-      )}
+          )}
 
-      <div className={'nav-mask' + (navVisible ? ' active' : '')} onClick={toggleNavVisible}></div>
-    </>
+          <div
+            className={'nav-mask' + (navVisible ? ' active' : '')}
+            onClick={toggleNavVisible}
+          ></div>
```
Comment (danny-avila) [inline client/src/components/Nav/Nav.tsx]: make this 

```
        <div
          className={'nav-mask' + (navVisible ? ' active' : '')}
          onClick={toggleNavVisible}
        />
```

and remove the empty brackets as before

**Review by danny-avila — CHANGES_REQUESTED** [review]

**Comment by danny-avila** [comment]
Thanks @Berry-13 lgtm! I tested and works well, no typescript issues, just a few minor syntax issues

**Comment by danny-avila** [comment]
also feel free to start making branches on the main repo so we can run playwright tests on your PRs

**Comment by Berry-13** [comment]
> Thanks @Berry-13 lgtm! I tested and works well, no typescript issues, just a few minor syntax issues

Done! In the meantime, I've also made some improvements to make the code more readable

**Review by danny-avila — APPROVED** [review]

**Pull request merged**